components/LoadingStatus.jsx

```diff
@@ -0,0 +1,17 @@
+export const loadingStates = {
+  EMPTY: 'empty',
+  LOADING: 'loading',
+  LOADED: 'loaded',
+  ERROR: 'error',
+};
+
+export default function LoadingStatus({ loadingStatus }) {
```

Comment:
Having loading is a really good idea. How fast something loads isn't always in our control so it's a good idea to do things like this to make page loading a slightly better experience.